README.md

```diff
@@ -650,7 +650,7 @@ They display information about object files and can be used to view an executabl
 # The packaging process
 Even if you are not interested in RPM, we would like to suggest to read the [RPM Packaging Tutorial](http://www.mac-vicar.eu/tutorials/rpm-packaging/index.html) since it gives a good overview and introduction.
 
-Then see the following system specific chapters here according to your interest.
+Then see the following distribution-specific chapters here according to your interest.
```

Comment:
Not sure about this one. It's not only distribution specific, but system. For example pkgsrc, homebrew etc could be added.

Reply:
Good point, I reverted to system-specific.